.github/workflows/label-analysis.yml

```diff
@@ -59,3 +66,5 @@ jobs:
             echo "Skip QA labels are not correct on PR ${{github.event.number}}. You must set 'qa/skip-qa' with exacly 1 justification, 'qa/done' or 'qa/no-code-change'"
             exit 1
           fi
+        env:
+          LABELS: ${{ steps.pr-labels.outputs.MILESTONE }}
```

Comment:
Same question here

Reply:
Thx, fixed in [9718176](https://github.com/DataDog/datadog-agent/pull/22657/commits/97181767da326ad97df91a600953d2974e0bf4af) !